carsus/io/output/base.py

```diff
@@ -185,6 +186,18 @@ def _create_metastable_flags(levels, lines,
 
     @staticmethod
     def _create_einstein_coeff(lines):
+        """
```

Comment:
Thanks a lot for adding a docstring. We use the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) docstring format in TARDIS and Carsus. Numpydoc has a References section where you can put the link and you could make a Notes section where you put the equations.